security/tailscale/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=       tailscale
+PLUGIN_VERSION=    1.0
```

Comment:
```suggestion
PLUGIN_VERSION=    1.0.d
PLUGIN_DEVEL=        yes
```

just for the moment mark this as devel only

Reply:
@fichtner thanks :)